packages/shared/config/environment.js

```diff
@@ -0,0 +1,18 @@
+// External dependencies
+const { join } = require('path');
+
+// Internal dependencies
+const { supportedEnvs } = require('../src/utils/constants');
+
+let path;
+const filename = '.env.';
+
+if (process.env.NODE_ENV === supportedEnvs.PRODUCTION) {
+  path = `/${filename}${process.env.NODE_ENV}`;
+} else {
+  path = join(__dirname, '../../..', filename);
```

Comment:
when we are not using production we our file will be **.env.** I think it should be .env

Reply:
That's right, the value of variable `path` (on line 13) will be `<monorepo_directory>/.env` on dev and test environment :)